standards.py

```diff
@@ -7,6 +7,12 @@
 import argparse
 import datetime
 
+"""
```

Comment:
Does this belong in this PR? :-)

Seems unrelated.

Reply:
yeah true that's noise, I'll create a separate PR for it